### ISSUE TITLE
Wrap an end-to-end pipeline in a clearml task an execute it remotely

### ISSUE THREAD

**Issue by alan02-dfki** [connected-intelligent-systems/forcateri#35]
- [x] Write a pipeline that works end-to-end. It should not be parametrized, yet. It should ingest a ClearML dataset, train a model, produce some predictions, compute some metrics and generate some matplotlib plots. Keep the number of rooms limited. The plots can be prediction vs. ground truth but also model performance depending on offset or $t_0$. All of this can happen locally.
- [x] Create a separate script creating a task that would execute the above pipeline and enqueue it to the `"default"`-queue. When computed, upload the tabular metrcs a csv files using ClearML's [artifact system](https://clear.ml/docs/latest/docs/clearml_sdk/task_sdk/#artifacts) and the scalar metrics using the [metric reporting](https://clear.ml/docs/latest/docs/clearml_sdk/task_sdk/#scalars). This can happen directly in the `ResultReporter` class, for now. We can loosen the dependency on ClearML at a later point.
- [x] Once the pipeline works remotely, you can parametrize it (e.g. through argparse) and make sure that the parameters can be conveniently set from the ClearML UI. Conveniently, here means that you should avoid parsing lists or dictionaries from strings, wherever possible, even if it comes at the cost of flexibility.
- [x] Once parametrized, you can do some manual (clone, adjust parameters and enqueue via UI) or semi-manual (clone, adjust parameters and enqueue via SDK) hyperparameter tuning on the existing model. Try comparing the results' plots or metrics via the UI and improve the reporting to make this comparison more convenient and more informative as you go.
